src/vm/eventtrace.cpp

```diff
@@ -5247,13 +5247,12 @@ VOID ETW::MethodLog::MethodJitted(MethodDesc *pMethodDesc, SString *namespaceOrC
         {
             // The call to SendMethodILToNativeMapEvent assumes that the debugger's lazy
             // data has already been initialized.
+			if (g_pDebugInterface != NULL)
+			{
+				g_pDebugInterface->InitializeLazyDataIfNecessary();
```

Comment:
Same as above, a future implementor should be forced to explicitly consider what happens if they set g_pDebugInterface = NULL. Losing the ILToNative events implicitly seems bad.